mesonbuild/modules/i18n.py

```diff
@@ -85,6 +86,10 @@ def merge_file(self, state, args, kwargs):
         ]
         if datadirs:
             command.append(datadirs)
+            
```

Comment:
__[Flake8]__ [W293] blank line contains whitespace ([link](https://sider.review/gh/repos/19784232/pulls/5300#issue-3997343))

<sub>posted by Sider</sub>

Reply:
Fixed and force-pushed a fix.